qa-tests-backend/src/main/proto

```diff
@@ -0,0 +1 @@
+/home/janisz/go/src/github.com/stackrox/stackrox/proto
```

Comment:
I'm not sure this will work properly. We should have a relative symlink.

Otherwise we can just add the creation of symlink (or directory copy) in Makefile.